### PR TITLE
Fix for open api filter 

### PR DESCRIPTION
**Pull request by Ralls0**
# Description

Fix oas filter

### CODE REVIEW

**Review by giorio94 — APPROVED** [review]
/lgtm

File: qlkube/src/decorateOpenapi.js [hunk 1]
```diff
@@ -2,14 +2,24 @@ const { apiGroups } = require('./apiGroups');
...
+
 function decorateOpenapi(oas) {
   let result = true;
   let keys = Object.keys(oas.paths);
   let removeKeys = keys.filter(key => {
     result = true;
     const paths = apiGroups.paths;
     paths.forEach(path => {
-      result &&= key.indexOf(path) == -1;
+      result = result && key.indexOf(path) === -1;
```
Comment (QcFe) [inline qlkube/src/decorateOpenapi.js:22]:
```suggestion
      result = result && !key.includes(path);
```
I'd suggest something similar to increase readability, also below

**Review by QcFe — APPROVED** [review]
Just a minor comment

File: qlkube/src/decorateOpenapi.js (hunk 1 above)
Comment (QcFe) [reply qlkube/src/decorateOpenapi.js:22]: (to be tested clearly)

**Pull request merged**